### DISCUSSION TITLE
Feature request to publish goal points in Map panel

### DISCUSSION THREAD

**Discussion by IvDmNe** [foxglove/community#920] [Q&A]
For now, we can publish robot goal points in 3D panel. For robots, operating in small environments, it is very useful. When we have larger environments, such as cities/fields/roads/etc, Map panel would be more convenient to set goal points that far away for many km from our robot/autonomous vehicle.

**Comment by jtbandes** [comment]
What message type and coordinate system would you use for this?

**Reply by IvDmNe** [reply]
It can be [LocationFix](https://github.com/foxglove/schemas/blob/main/schemas/ros1/LocationFix.msg) in WGS84 coordinate system, though it lacks orientation information
